R/result.R

```diff
@@ -1,36 +1,364 @@
+#' Version of R6 with heritable static/class methods and attributes
```

Comment:
This is really cool, Ed. Honestly I think this and the `Serializable` deserve their own package, or at least issue in https://github.com/r-lib/R6 .

Reply:
Maybe `Serializable` doesn't necessarily belong in R6, but heritable classmethods are such a common tool in Python that I'm surprised they don't already exist in R6.